The job **`x86_64-gnu-tools`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/6114300112/job/16595578723) [(plain)](https://github.com/rust-lang/rust/commit/5c6380f82e7aedb904eabeb429f5d5f7d84449bc/checks/16595578723/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
GITHUB_ACTION=__run_7
GITHUB_ACTIONS=true
GITHUB_ACTION_REF=
GITHUB_ACTION_REPOSITORY=
GITHUB_ACTOR=notriddle
GITHUB_API_URL=https://api.github.com
GITHUB_BASE_REF=master
GITHUB_ENV=/home/runner/work/_temp/_runner_file_commands/set_env_731bf7f2-3a03-43c7-9baf-0cde9638352f
GITHUB_EVENT_NAME=pull_request
GITHUB_EVENT_NAME=pull_request
GITHUB_EVENT_PATH=/home/runner/work/_temp/_github_workflow/event.json
GITHUB_GRAPHQL_URL=https://api.github.com/graphql
GITHUB_HEAD_REF=notriddle/rustdoc-fe-cleanup
GITHUB_JOB=pr
GITHUB_PATH=/home/runner/work/_temp/_runner_file_commands/add_path_731bf7f2-3a03-43c7-9baf-0cde9638352f
GITHUB_REF=refs/pull/115655/merge
GITHUB_REF_NAME=115655/merge
GITHUB_REF_PROTECTED=false
---
GITHUB_SERVER_URL=https://github.com
GITHUB_SHA=23e1fcf79295e72946d4c0387d6cb70fb9f292fe
GITHUB_STATE=/home/runner/work/_temp/_runner_file_commands/save_state_731bf7f2-3a03-43c7-9baf-0cde9638352f
GITHUB_STEP_SUMMARY=/home/runner/work/_temp/_runner_file_commands/step_summary_731bf7f2-3a03-43c7-9baf-0cde9638352f
GITHUB_TRIGGERING_ACTOR=notriddle
GITHUB_WORKFLOW_REF=rust-lang/rust/.github/workflows/ci.yml@refs/pull/115655/merge
GITHUB_WORKFLOW_SHA=23e1fcf79295e72946d4c0387d6cb70fb9f292fe
GITHUB_WORKSPACE=/home/runner/work/rust/rust
GOROOT_1_18_X64=/opt/hostedtoolcache/go/1.18.10/x64
---
FF (100/108)
........   (108/108)


/checkout/tests/rustdoc-gui/docblock-code-block-line-number.goml docblock-code-block-line-number... FAILED
[ERROR] (around line 63): JS errors occurred: Error: ReferenceError: blurHandler is not defined
    at HTMLAnchorElement.settingsBlurHandler (file:///checkout/obj/build/x86_64-unknown-linux-gnu/test/rustdoc-gui/doc/static.files/settings-de11bff964e9d4e5.js:17:1651)
/checkout/tests/rustdoc-gui/notable-trait.goml notable-trait... FAILED
[ERROR] (around line 252): JS errors occurred: Error: ReferenceError: blurHandler is not defined
    at HTMLAnchorElement.settingsBlurHandler (file:///checkout/obj/build/x86_64-unknown-linux-gnu/test/rustdoc-gui/doc/static.files/settings-de11bff964e9d4e5.js:17:1651)
/checkout/tests/rustdoc-gui/pocket-menu.goml pocket-menu... FAILED
[ERROR] (around line 23): JS errors occurred: Error: ReferenceError: blurHandler is not defined
    at HTMLAnchorElement.settingsBlurHandler (file:///checkout/obj/build/x86_64-unknown-linux-gnu/test/rustdoc-gui/doc/static.files/settings-de11bff964e9d4e5.js:17:1651)
/checkout/tests/rustdoc-gui/search-filter.goml search-filter... FAILED
[ERROR] (around line 76): JS errors occurred: Error: ReferenceError: blurHandler is not defined
    at HTMLAnchorElement.settingsBlurHandler (file:///checkout/obj/build/x86_64-unknown-linux-gnu/test/rustdoc-gui/doc/static.files/settings-de11bff964e9d4e5.js:17:1651)
/checkout/tests/rustdoc-gui/setting-auto-hide-content-large-items.goml setting-auto-hide-content-large-items... FAILED
[ERROR] (around line 29): JS errors occurred: Error: ReferenceError: blurHandler is not defined
    at HTMLAnchorElement.settingsBlurHandler (file:///checkout/obj/build/x86_64-unknown-linux-gnu/test/rustdoc-gui/doc/static.files/settings-de11bff964e9d4e5.js:17:1651)
/checkout/tests/rustdoc-gui/setting-auto-hide-item-methods-docs.goml setting-auto-hide-item-methods-docs... FAILED
[ERROR] (around line 26): JS errors occurred: Error: ReferenceError: blurHandler is not defined
    at HTMLAnchorElement.settingsBlurHandler (file:///checkout/obj/build/x86_64-unknown-linux-gnu/test/rustdoc-gui/doc/static.files/settings-de11bff964e9d4e5.js:17:1651)
/checkout/tests/rustdoc-gui/setting-auto-hide-trait-implementations.goml setting-auto-hide-trait-implementations... FAILED
[ERROR] (around line 25): JS errors occurred: Error: ReferenceError: blurHandler is not defined
    at HTMLAnchorElement.settingsBlurHandler (file:///checkout/obj/build/x86_64-unknown-linux-gnu/test/rustdoc-gui/doc/static.files/settings-de11bff964e9d4e5.js:17:1651)
/checkout/tests/rustdoc-gui/setting-go-to-only-result.goml setting-go-to-only-result... FAILED
[ERROR] (around line 30): JS errors occurred: Error: ReferenceError: blurHandler is not defined
    at HTMLAnchorElement.settingsBlurHandler (file:///checkout/obj/build/x86_64-unknown-linux-gnu/test/rustdoc-gui/doc/static.files/settings-de11bff964e9d4e5.js:17:1651)
/checkout/tests/rustdoc-gui/settings.goml settings... FAILED
[ERROR] (around line 26): JS errors occurred: Error: ReferenceError: blurHandler is not defined
    at HTMLAnchorElement.settingsBlurHandler (file:///checkout/obj/build/x86_64-unknown-linux-gnu/test/rustdoc-gui/doc/static.files/settings-de11bff964e9d4e5.js:17:1651)
/checkout/tests/rustdoc-gui/theme-change.goml theme-change... FAILED
[ERROR] (around line 12): JS errors occurred: Error: ReferenceError: blurHandler is not defined
    at HTMLAnchorElement.settingsBlurHandler (file:///checkout/obj/build/x86_64-unknown-linux-gnu/test/rustdoc-gui/doc/static.files/settings-de11bff964e9d4e5.js:17:1651)
FFFFFFFFError: ()
  local time: Thu Sep  7 20:34:20 UTC 2023
  network time: Thu, 07 Sep 2023 20:34:20 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/src/librustdoc/html/markdown.rs
+++ b/src/librustdoc/html/markdown.rs
@@ -1574,7 +1574,6 @@ fn init_id_map() -> FxHashMap<Cow<'static, str>, usize> {
     map.insert("crate-search-div".into(), 1);
     // This is the list of IDs used in HTML generated in Rust (including the ones
     // used in tera template files).
-    map.insert("mainThemeStyle".into(), 1);
     map.insert("themeStyle".into(), 1);
     map.insert("settings-menu".into(), 1);
     map.insert("help-button".into(), 1);

--- a/src/librustdoc/html/templates/page.html
+++ b/src/librustdoc/html/templates/page.html
@@ -15,8 +15,7 @@
     <link rel="stylesheet" {#+ #}
           href="{{static_root_path|safe}}{{files.normalize_css}}"> {# #}
     <link rel="stylesheet" {#+ #}
-          href="{{static_root_path|safe}}{{files.rustdoc_css}}" {#+ #}
-          id="mainThemeStyle"> {# #}
+          href="{{static_root_path|safe}}{{files.rustdoc_css}}"> {# #}
     {% if !layout.default_settings.is_empty() %}
     <script id="default-settings" {#+ #}
       {%~ for (k, v) in layout.default_settings ~%}

